Practice/kruglov/home_task_4/PyLec4T3-4.py

```diff
@@ -0,0 +1,38 @@
+# 3 poor solution
+# d = [0, 3, 24, 2, 3, 7]
+# b = []
+
+# i = 0
+# s = len(d)
+# while i < s:
+#    c = min(d)
+#    b.append(c)
+#    d.pop(d.index(c))
+#    i += 1
+
+# print(b)
+
+# 3.1
+list1 = [0, 3, 24, 2, 3, 7]
+el = 0
+len_list = len(list1)
+while el < len_list:
+    min_ind = list1.index(min(list1[el:]), el)
+    min_val = list1.pop(min_ind)
```

Comment:
For lines 21-25: there's more pythonic way to perform this swap: 
```python
list1[min_ind], list1[el] = list1[el], list1[min_ind]
```

Reply:
This is some kind of magic!

Reply:
Fixed

Reply:
It's not magic, it's Python:)